x/shield/legacy/v2/mixed_coins.go

```diff
@@ -0,0 +1,71 @@
+package v2
```

Comment:
do we need this file? I don't think we need any calculations during the migration.

Reply:
As explained on the comment above.